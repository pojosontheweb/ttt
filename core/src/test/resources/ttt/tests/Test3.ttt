<%@ page import="com.xyz.myapp.Foo" %>
<%@ page extends="java.io.Serializable" %>
<%!
    Foo foo;
%>
this is a <%= foo.getBar() %> template