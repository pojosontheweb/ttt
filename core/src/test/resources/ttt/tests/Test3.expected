package ttt.tests;

import com.xyz.myapp.Foo;

public class Test3 extends com.pojosontheweb.ttt.Template implements java.io.Serializable {

	private final Foo foo;

	public Test3(Foo foo) {
		this.foo = foo;
	}

	@Override
	public void render(java.io.Writer out) throws java.io.IOException {
		write(out, "this is a " );
		write(out,  foo.getBar()  );
		write(out, " template" );
	}
}
